src/App.tsx

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
-import Dashboard from './components/dashboard';
-import Login from './components/login';
+import "./App.css";
```

Comment:
**suggestion**: Consider adding PropTypes or TypeScript interfaces to define expected props for better type safety.